src/docs/07-operation-guide/01-setup.md

```diff
@@ -156,6 +160,31 @@ Then you should see the logs like below
 cadence_1        | {"level":"info","ts":"2021-05-07T18:43:07.869Z","msg":"First loading dynamic config","service":"cadence-frontend","key":"frontend.visibilityListMaxQPS,domainName:sample,clusterName:primary","value":"10000","default-value":"10","logging-call-at":"config.go:93"}
 ```
 
+#### Config store client
+
+You can set the `dynamicconfig` client in the static configuration to `configstore` in order to store config changes in a database, as shown below. 
+```yaml
+dynamicconfig:
+  client: configstore
+  configstore:
+    pollInterval: "10s"
+    updateRetryAttempts: 2
+    FetchTimeout: "2s"
+    UpdateTimeout: "2s"
+```
+
+If you are still using the deprecated config `dynamicConfigClient` like below, you need to replace it with the new `dynamicconfig` as shown above to use `configstore` client.
+```yaml
+dynamicConfigClient:
+  filepath: "/etc/cadence/config/dynamicconfig/config.yaml"
+  pollInterval: "10s"
+```
+
+After changing the client to `configstore` and restarting Cadence, you can manage dynamic configs using `cadence admin config` CLI commands.
```

Comment:
in this case, it is worth to mention that custom values needs to be set once again as keys/values are not migrated from file to a database

Reply:
Thanks for the comment. I've added a reminder here about migrating custom configs.